.github/workflows/main.yaml

```diff
@@ -25,59 +31,33 @@ jobs:
               echo "yes!" > NEED_UPDATE_MINIMAL
```

Comment:
`NEED_UPDATE_MINIMAL` is no longer in use.

Reply:
removed.